panko_serializer.gemspec

```diff
@@ -31,4 +31,5 @@ Gem::Specification.new do |spec|
   spec.extensions << "ext/panko_serializer/extconf.rb"
 
   spec.add_dependency "oj", "~> 3.10.0"
+  spec.add_dependency "activesupport"
```

Comment:
Since the resolver uses a bunch of ActiveSupport core_ext, it's better to add it to the dependencies.

Reply:
It was always a question for me, whether I want a direct dependency on rails for Panko since it can serialize plain ruby objects. but to be honest with me, it's 90% of the project, so ok. 👍

Reply:
Up to you, we can also try to eliminate the dependency. 

`camelize` for instance is very easy to get rid of, `singularize` however it's quite trickier.

But maybe there's a way to have this code only called in Rails context? I don't know.